src/features/emails/hooks/useSendTestEmail.ts

```diff
@@ -1,19 +1,42 @@
 import { useState } from 'react';
 
+import { useUser } from 'utils/hooks/useFocusDate';
+import { useApiClient, useNumericRouteParams } from 'core/hooks';
+
 export default function useSendTestEmail() {
+  const apiClient = useApiClient();
   const [isLoading, setIsLoading] = useState(false);
+  const [isPostSend, setIsPostSend] = useState(false);
```

Comment:
I wish this state was named differently, I have tried for a while and I don't understand what this means - thinking either that PostSend is a type of object or something and this says if something is a PostSend or not, in that case maybe call it isCorrectType or something? OR post here means after, and in that case maybe call it wasSent or isAfterSending or something?

Reply:
You're right! I renamed it to `emailWasSent`.